### ISSUE TITLE
disable info in problems view

### ISSUE THREAD

**Issue by OverDrone** [nodj/AutoDeriv#23]
Hello!
Each time eclipse starts up Autoderiv plugin adds marker `Added rule: ...`  with type=info which goes to Problems view. Each time I have to manually delete those rows. Is there an option to disable those infos? I tried to disable all checkboxes related to logging in autoderiv settings, it didn't help
Update: 
Found setting `Add markers in .derived file...`. Very strange idea to show info marker in `Problems view`. Having entries in `Problems view` basically means something wrong with my code/project setup. I'm ok with warnings/errors for .derived files if it's something wrong with their contents, but why show `info` when everything is ok and working as expected. I specifically installed this plugin to add gradle `build` folder to derived which sometimes resets after rebuild without this plugin. That means I expect `build` folder to be always derived, no reason to tell me this every eclipse startup.

In other words separate `Add markers in .derived file...` setting into two `Add errors/warnings markers in .derived file...` and `Add info markers in .derived file...` or even remove second checkbox completely and change behavior from info/warning/error to warning/error. Because as I said I don't want to see infos, but I want to see errors/warnings for `.derived` files if there is something wrong with them.
